flexget/plugins/parsers/parser_guessit.py

```diff
@@ -229,7 +263,7 @@ def parse_series(self, data, **kwargs):
             episode = episode[0]
         date = guess_result.get('date')
         quality = self._quality(guess_result)
-        proper_count = self._proper_count(guess_result)
+        proper_count = self._proper_count(guess_result, )
```

Comment:
Why the comma?

Reply:
Nice catch. Small typo. Committed a fix.